tests/other/metadce/hello_libcxx_O2_fexceptions_DEMANGLE_SUPPORT.funcs

```diff
@@ -533,6 +532,7 @@ $pad
 $pop_arg
 $pop_arg_long_double
 $printf_core
+$pthread_cond_wait
```

Comment:
This seems to be the result of no using `--whole-archive` around libc++abi..    I'm not sure what to make of it .. but I'm pretty sure we don't want `--whole-archive`

Reply:
I wonder if it happens to cause a different order of functions, and then different function sizes (LEBs of call indexes change), and it's just enough to cause an inlining difference...

Maybe just verify locally that the wasm size hasn't changed much? (the metadce test should, but would be good to know if this is just under the size of the slack we allow to change)